firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -50,6 +52,7 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
 
     private var showDeletion = false
     private var sectionsToDisplay: [SearchSettingsTableViewController.Section] = []
+    private var searchSettingsState: SearchSettingsState
```

Comment:
```suggestion
    private var searchSettingsState: SearchSettingsState?
```
This should be optional, and then assigned in `newState`